resources/assets/js/app/views/topics/topicCategoryItemForSelector.js

```diff
@@ -3,15 +3,21 @@ var Marionette = require('backbone.marionette');
 module.exports = Marionette.ItemView.extend({
     template: 'topicCategoryItemForSelector',
 
-    onRender: function () {
-        console.log(this.collection, 'render');
-    },
-
     collectionEvents: {
-        'sync': 'render'
+        'sync': 'reRender'
     },
 
-    initialize: function (options) {
-    }
+    reRender: function () {
+        var self = this;
 
+        this.collection.forEach (function (model, index) {
+            if (self.options.categoryId == model.get('id')) {
+                model.attributes.selectedItem = 'selected'
+            } else {
+                model.attributes.selectedItem = ''
+            }
+        });
```

Comment:
use appropriate key from topicModel and set need dropdown value depends on this, instead of this